client-v2/src/components/FrontsEdit/FrontContent.tsx

```diff
@@ -0,0 +1,220 @@
+import React from 'react';
```

Comment:
These files are large, but most of these lines haven't been changed -- they're the result of `FrontContainer.tsx` being split out into `FrontContent.tsx` and `FrontContainer.tsx`.